communications/meetings.md

```diff
@@ -0,0 +1,56 @@
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Preferably 1 work-day before the meeting, initiate one discussion per agenda item.
+    - The discussion can be a Slack thread, GitHub issue, Asana task, Google Docs, or anything with an interactive "comments" component.
+    - For a recurring meeting with past sessions, use the ones from the previous meeting session as a starting point. Remove ones no longer needed.
+    - For a net-new meeting, the organizer(s) should define the initial scope of items.
+    - Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
+- In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant.
+    - For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each _has done_, _has been doing_, and _plans to do_. Give everyone else a sense of _where you are_ and _the direction you're heading to_. This helps others to determine if there are potential directional conflicts/blocks.
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+
+## During A Meeting
+
+Objectives:
+- Maximize the gain of synchronous/in-person/voice communication to resolve items.
+- Record as many details as possible for future context.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+- Control time usage.
+
+Guide:
+- Demonstrate issues, instead of just abstractly talking about them.
+- Aggressively control the timebox of each item. If the item is resolved sooner than the tentative timebox, skip ahead promptly, leave the jokes and fun talks for some other time.
+- If certain items cannot be resolved within the timebox, record all reasons and context into the per-item discussion formed in [section: Before A Meeting](#before-a-meeting) and move on.
+- Proactively record notes in the per-item discussion formed in [section: Before A Meeting](#before-a-meeting).
+
+## After A Meeting
+
+Objectives:
+- Have clear takeaways.
+- Organize meeting notes to be as comprehensive as possible, for both non-participants and future self.
+- Determine suitable follow-ups for leftovers.
+
+Guide:
+- Amend per-item discussion with afterthoughts.
```

Comment:
Can combine amend & summarize into one point. Suggestion:

```
* Amend, summarize and derive actionable item for each agenda item
    * It may be a good idea to use a more distinguishable section for the summaries and actionable, highly dependent on what tool is used for the discussions (WIP: need suggestions).

```